### PR TITLE
Feature/cypress

### PR DESCRIPTION
**Pull request by Swatzell**
This PR will setup initial cypress testing on our site by:

- establishing a mock data fixture for cypress to work off of.
- writing intercepts for fetch apis
- writing passing IT blocks for testing of initial basic functionality of the site.

### CODE REVIEW

File: cypress/fixtures/users.json
Comment (LISims88) [inline cypress/fixtures/users.json:1]: I have aquestion about the use of this data. When and where are we using this information or is it just mock data?

Comment (Swatzell) [reply cypress/fixtures/users.json:1]: When I installed cypress it automatically made some fixture json files and that was one of them. I removed the other but just forgot to remove that one. I'll take it out when I add some more testing tomorrow.

**Review by LISims88 — COMMENTED** [review]
Great use of Cypress Testing can't wait to see it in action.

File: cypress/fixtures/users.json
Comment (LISims88) [reply cypress/fixtures/users.json:1]: Thank you for your answer.

**Pull request merged**